### PR TITLE
xcpretty: update rubocop to 0.72.0.

### PR DESCRIPTION
**Pull request by barakwei**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -223,10 +222,16 @@ Style/StringLiteralsInInterpolation:
 Style/SymbolProc:
   Enabled: false
 
-Style/TrailingComma:
-  Enabled: true
+Style/TrailingCommaInArguments:
+  Enabled: false
```
Comment (DekelAvrahami) [inline .rubocop.yml]: `true`. Same below.

File: bin/xcpretty [hunk 2]
```diff
@@ -49,11 +50,11 @@ OptionParser.new do |opts|
   end
   opts.on("-r", "--report FORMAT or PATH", "Run FORMAT or PATH reporter",
           "  Choices: #{report_formats.keys.join(', ')}") do |format|
-    if report_formats.key?(format)
-      report_classes << report_formats[format]
-    else
-      report_classes << XCPretty.load_custom_class(format)
-    end
+    report_classes << if report_formats.key?(format)
```
Comment (DekelAvrahami) [inline bin/xcpretty:53]: Maybe use `?:` operator to use less vertical space.

File: lib/xcpretty/formatters/formatter.rb [hunk 3]
```diff
@@ -104,11 +105,11 @@ def use_unicode?
     # Will be printed by default. Override with '' if you don't want summary
     def format_test_summary(executed_message, failures_per_suite)
       failures = format_failures(failures_per_suite)
-      if failures.empty?
-        final_message = green(executed_message)
-      else
-        final_message = red(executed_message)
-      end
+      final_message = if failures.empty?
```
Comment (DekelAvrahami) [inline lib/xcpretty/formatters/formatter.rb:108]: Maybe use `?:` operator to use less vertical space.

File: lib/xcpretty/formatters/formatter.rb [hunk 4]
```diff
@@ -176,18 +177,18 @@ def format_failures(failures_per_suite)
...
+      output += "\n  #{cyan(failure[:file_path])}"
       return output if snippet.contents.empty?
 
       output += "\n  ```\n"
-      if @colorize
-        output += Syntax.highlight(snippet)
-      else
-        output += snippet.contents
-      end
+      output += if @colorize
```
Comment (DekelAvrahami) [inline lib/xcpretty/formatters/formatter.rb]: Maybe use `?:` operator to use less vertical space.

File: lib/xcpretty/syntax.rb [hunk 5]
```diff
@@ -12,20 +14,22 @@ module XCPretty
...
       highlight_with_formatter(snippet, Rouge::Formatters::HTML.new)
     end
 
     def self.highlight_with_formatter(snippet, formatter)
-      if snippet.file_path.include?(':')
-        filename = snippet.file_path.rpartition(':').first
-      else
-        filename = snippet.file_path
-      end
+      filename = if snippet.file_path.include?(':')
```
Comment (DekelAvrahami) [inline lib/xcpretty/syntax.rb:28]: Maybe use `?:` operator to use less vertical space.

**Review by DekelAvrahami — CHANGES_REQUESTED** [review]
Fix and squash.

File: .rubocop.yml (hunk 1 above)
Comment (barakwei) [reply .rubocop.yml]: done

File: lib/xcpretty/formatters/formatter.rb (hunk 4 above)
Comment (barakwei) [reply lib/xcpretty/formatters/formatter.rb]: this works

File: bin/xcpretty (hunk 2 above)
Comment (barakwei) [reply bin/xcpretty:53]: rubocop doesn't like multiline ternary

File: lib/xcpretty/formatters/formatter.rb (hunk 3 above)
Comment (barakwei) [reply lib/xcpretty/formatters/formatter.rb:108]: rubocop doesn't like multiline ternary

File: lib/xcpretty/syntax.rb (hunk 5 above)
Comment (barakwei) [reply lib/xcpretty/syntax.rb:28]: rubocop doesn't like multiline ternary

**Comment by barakwei** [comment]
fixed what I could. rubocop here is pretty strict. I like it.

**Review by DekelAvrahami — APPROVED** [review]
LGTM.

**Pull request merged**